lib/platform/github/index.ts

```diff
@@ -199,14 +219,26 @@ export async function initRepo({
     hostType: PlatformId.Github,
     url: defaults.endpoint,
   });
-  config.isGhe = URL.parse(defaults.endpoint).host !== 'api.github.com';
   config.renovateUsername = renovateUsername;
   [config.repositoryOwner, config.repositoryName] = repository.split('/');
   let repo: GhRepo;
   try {
+    let infoQuery = repoInfoQuery;
+
+    if (
+      config.isGhe &&
+      (!config.gheVersion ||
+        semverSatisfies(config.gheVersion, '>=3.1.0 <3.1.8') ||
+        semverSatisfies(config.gheVersion, '>=3.0.0 <3.0.16'))
+    ) {
+      // Changelog: https://docs.github.com/en/github-ae@latest/graphql/overview/changelog?query=#schema-changes-for-2021-08-03
+      // Release schedule: https://enterprise.github.com/releases
```

Comment:
Are you sure that applies to GHES?

Reply:
e.g. if we go to https://docs.github.com/en/enterprise-server@3.2/graphql/reference/objects and search for `hasIssuesEnabled` you find it, but you don't find `autoMergeAllowed`

Reply:
I don't know Github's release process well, but I've noticed we have same changelog for different minor lineages:
https://docs.github.com/en/enterprise-server@3.2/graphql/overview/changelog#schema-changes-for-2021-08-03
https://docs.github.com/en/enterprise-server@3.1/graphql/overview/changelog#schema-changes-for-2021-08-03
https://docs.github.com/en/enterprise-server@3.0/graphql/overview/changelog#schema-changes-for-2021-08-03

Maybe, my premise is false, and seems like I have no chance to check it quickly

Reply:
OK, let's try it. If we still get errors from users on the latest patch, we need to remove it altogether from GHE until we get clarity. I raised a support ticket to GitHub asking them, but we'll need to release something before they are likely to respond to me